configure.ac

```diff
@@ -632,7 +632,7 @@ echo
 echo '* Checking for apt_pkg library used by: dpkginfo '
 PKG_CHECK_MODULES([apt_pkg], [libapt-pkg >= 0.0],[],[
 SAVE_LIBS=$LIBS
-AC_SEARCH_LIBS([pkgInitConfig],[apt-pkg],[
+AC_SEARCH_LIBS([pkgVersion],[apt-pkg],[
```

Comment:
Ok, so I was confused a little bit. Let me start again.

When I take this PR and I run `./confgen.sh ; git diff` it is not clean.

What we perhaps need to do is to patch `ac_probes/libs/apt_pkg` like:
```
-functions_req=(pkgInitConfig pkgInitSystem)
+functions_req=(pkgVersion)
```

and then run ./confgen.sh on Fedora. That way we ensure that confgen is always clean.

However, it is not clear to me whether it is enough for Debian build to pass. Can you please re-check on your debian vm that build passes?